e2e/cypress/integration/system_console/search_teams_spec.js

```diff
@@ -38,29 +38,31 @@ describe('Search teams', () => {
         cy.get('[data-test=search-input]').type(displayName + '{enter}');
 
         // page 1 has PAGE_SIZE items
-        cy.get('[data-test=team-display-name]').as('displayNames').should('have.length', PAGE_SIZE);
+        cy.get('[data-test=team-display-name]').should('have.length', PAGE_SIZE);
 
         // page 2 has 2 items
         cy.get('[data-test=page-link-next]').should('be.enabled').click();
-        cy.get('@displayNames').should('have.length', 2);
+        cy.get('[data-test=team-display-name]').should('have.length', 2);
     });
 
     it('clears the results when "x" is clicked', () => {
         const displayName = uuid();
         cy.apiCreateTeam('team-search', displayName);
         cy.get('[data-test=search-input]').as('searchInput').type(displayName + '{enter}');
-        cy.get('[data-test=team-display-name]').as('displayNames').should('have.length', 1);
+        cy.get('[data-test=team-display-name]').should('have.length', 1);
+
         cy.get('[data-test=clear-search]').click();
         cy.get('@searchInput').should('be.visible').and('have.text', '');
-        cy.get('@displayNames').should('have.length', PAGE_SIZE);
+        cy.get('[data-test=team-display-name]').should('have.length', PAGE_SIZE);
     });
 
     it('clears the results when the search term is deleted with backspace', () => {
         const displayName = uuid();
         cy.apiCreateTeam('team-search', displayName);
         cy.get('[data-test=search-input]').as('searchInput').type(displayName + '{enter}');
-        cy.get('[data-test=team-display-name]').as('displayNames').should('have.length', 1);
+        cy.get('[data-test=team-display-name]').should('have.length', 1);
+
         cy.get('@searchInput').type('{selectall}{del}');
-        cy.get('@displayNames').should('have.length', PAGE_SIZE);
+        cy.get('[data-test=team-display-name]').should('have.length', PAGE_SIZE);
     });
 });
```

Comment:
Since the System Console > Teams is an Enterprise level feature, can you please move this test to the following folder:
e2e > cypress > integration > enterprise > system_console

Reply:
Moved.